frozendict/__init__.py

```diff
@@ -1,6 +1,7 @@
 import collections
-import operator
-import functools
+
+
+iteritems = getattr(dict, 'iteritems', dict.items)
```

Comment:
I was assuming it'd be inside the `if self._hash is None:` block, in which case it'd only ever get assigned once? (Or not at all, if the `__hash__` function is never called)